### PR TITLE
Send email to organizations with the exports split revenue #220

### LINKED ISSUE #220: Send email to organizations with the exports split revenue
On Django command that exports split revenue for all organization, 1 file per organization, we need to add a parameter that allows the operator to send an email per organization with the file link.

Each file needs to have an hash random has suffix of the file name, with a length of 10 characters.

Example:
For organization INA the file will be `report_split_revenue_INA_20231213110407_abcdefghij.xlsx`

Document on the README that the installer needs to use the S3 Django storage with a public storage configuration.

The `From` of the emails must be configurable by a Django Setting.

### PR DESCRIPTION
**Pull request by Tiago-da-silva23**
This PR inserts the architecture to send emails when the file export per organization command is triggered, resolve #220.

### CODE REVIEW

File: apps/shared_revenue/management/commands/export_split_revenue_per_organizations.py
Comment (jamoqs) [inline apps/shared_revenue/management/commands/export_split_revenue_per_organizations.py:1]: The logic and the function to send_mail can't be in this file, need to stay in the scope system like SplitExportService class, then can be shared and imported on entire system.

File: apps/shared_revenue/services/split_export.py
Comment (jamoqs) [inline apps/shared_revenue/services/split_export.py:1]: Why i have to lines to generate one file instanced in a variable?

    file_name: str = self._generate_file_name(optional=kwargs)
    FileGenerator().generate_xlsx(file_name=file_name, sheets=split_sheets)

File: apps/shared_revenue/tasks.py
Comment (jamoqs) [inline apps/shared_revenue/tasks.py:1]: i have two logics to do same thing, but they are not shared. The sendmail things at Command file

**Review by jamoqs — CHANGES_REQUESTED** [review]

File: apps/shared_revenue/management/commands/export_split_revenue_per_organizations.py [hunk 1]
```diff
@@ -1,56 +1,83 @@
...
     How to use:
 
-        python manage.py export_split_revenue_per_organizations  {start_date} {end_date}
+        python manage.py export_split_revenue_per_organizations  {start_date} {end_date} {send_file}
 
         Exemple:
 
-            python manage.py export_split_revenue_per_organizations 2023-12-01 2024-01-01
-
+            python manage.py export_split_revenue_per_organizations 2023-12-01 2024-01-01 true
```
Comment (sandroscosta) [inline apps/shared_revenue/management/commands/export_split_revenue_per_organizations.py:28]: This command can be a bit more explicit.
`python manage.py export_split_revenue_per_organizations 2023-12-01 2024-01-01 --send-email`
As the intention is to send an email message to the organization.

File: apps/shared_revenue/management/commands/export_split_revenue_per_organizations.py [hunk 2]
```diff
@@ -1,56 +1,83 @@
...
         - start_date: YYYY-MM-DD
         - end_date: YYYY-MM-DD
+        - send_file: true / false
 
     How to use:
 
-        python manage.py export_split_revenue_per_organizations  {start_date} {end_date}
+        python manage.py export_split_revenue_per_organizations  {start_date} {end_date} {send_file}
 
         Exemple:
```
Comment (sandroscosta) [inline apps/shared_revenue/management/commands/export_split_revenue_per_organizations.py:26]: Small typo. **Example**

**Review by jamoqs — APPROVED** [review]

**Pull request merged**